requirements.txt

```diff
@@ -32,3 +32,6 @@ pre-commit==2.13.0
 
 flake8==3.9.2
 isort==5.9.1
+
+# for concept2sentence
+sibyl_tool==0.1.2
```

Comment:
Hi @fabriceyhc thanks for your changes. Please add this under the transformations folder.

Reply:
Will do. Just to confirm, I'll be removing it from `NL-Augmenter\requirements.txt` and creating my own `requirements.txt` in the `concept2sentence` folder. 

Also, I see that the build is failing for not having a test.json file. I thought it was optional? My transformation has some randomness in it that may not yield the same outputs every time so the tests are likely to fail since they're looking for exact match. How can we work around this? If necessary, I can make the outputs deterministic, but it undermines a diversifying goal of the transformation.

Reply:
Hello @fabriceyhc my apologies for the delayed reply! Yes - you are right about the hierarchy. You can temporarily add the test cases like [this](https://github.com/GEM-benchmark/NL-Augmenter/blob/main/transformations/diverse_paraphrase/test.json).